.travis.yml

```diff
@@ -27,6 +27,7 @@ install:
     'pandas>=0.18' 'matplotlib>=1.1.0' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1'
   - source activate env_name
   - pip install sphinx sphinx-bootstrap-theme coveralls 'ipython[all]==2.4.1'
+  - pip install https://github.com/qiita-spots/qiita-files/archive/master.zip --no-deps
```

Comment:
Why --no-deps and shouldn't this be automatically installed (added to setup.py)?